coordinator/core/src/test/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/GlobalAggregationCalculatorTest.kt

```diff
@@ -750,6 +731,19 @@ class GlobalAggregationCalculatorTest {
       val expectedAggregations: List<BlobsToAggregate>
     )
 
+    private fun assertOneToAggregationSizeUsesAggregationSize(
```

Comment:
Would `checkAggregationSizesBelowMaxAggregationSize` make more sense?

Reply:
hmmm, it also includes the max size, so below doesn't work
`checkAggregationSizesBelowAndIncludingMaxAggregationSize` ?

Reply:
`checkAggregationSizesNotExceedingMaxAggregationSize` is slightly shorter if we want to be super precise

Reply:
done